e2e_tests/tests/cluster/managed_cluster.py

```diff
@@ -110,14 +112,16 @@ def restart_proxy(self, wait_for_reconnect: bool = True) -> None:
                 pytest.fail(f"Agent didn't reconnect after {_i} seconds")
 
     def ensure_agent_ok(self) -> None:
-        agent_data = get_agent_data(conf.make_master_url())
+        admin = api_utils.admin_session()
```

Comment:
hm... no I think you're right, I don't think this file needs it.  I'll reduce to normal user permissions and see if that isn't enough.

I _think_ this was a side-effect of how long I spent refactoring it.  Some parts of the refactor are written with different mindsets.  In this file I think the mindset was "well this is a helper function and not an rbac test so we should just always use admin since it's guaranteed to have sufficient permissions" but that mindset eventually broke down when interacting with clusters created by more exotic means (because the singleton admin_session() might not work against the current master).

Nice catch, thank you.  I searched for other obviously unnecessary admin usages, let me know if you run across additional ones.